tooling/metamodel-generator/src/main/java/org/hibernate/jpamodelgen/annotation/CriteriaFinderMethod.java

```diff
@@ -69,9 +69,13 @@ public String getAttributeDeclarationString() {
 						.append(");");
 			}
 		}
+		String localSessionName = isReactiveSession() ? "_session" : sessionName;
+		if(isReactiveSession()) {
+			declaration.append("\n\treturn ").append(sessionName).append(".flatMap(").append(localSessionName).append(" -> {");
+		}
```

Comment:
I think our convention is to use `chain()` in preference to `flatMap()`.

Reply:
OK